vllm/executor/ray_gpu_executor.py

```diff
@@ -136,46 +138,38 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
-        all_args_to_update_environment_variables = []
-        for (node_id, _) in worker_node_and_gpu_ids:
-            all_args_to_update_environment_variables.append([{
-                "CUDA_VISIBLE_DEVICES":
-                ",".join(map(str, node_gpus[node_id])),
-                "VLLM_INSTANCE_ID":
-                VLLM_INSTANCE_ID,
-                "VLLM_TRACE_FUNCTION":
-                os.getenv("VLLM_TRACE_FUNCTION", "0"),
-            }])
+        all_args_to_update_environment_variables = [({
+            "CUDA_VISIBLE_DEVICES":
+            ",".join(map(str, node_gpus[node_id])),
+            "VLLM_INSTANCE_ID":
+            VLLM_INSTANCE_ID,
+            "VLLM_TRACE_FUNCTION":
+            os.getenv("VLLM_TRACE_FUNCTION", "0"),
+        }, ) for (node_id, _) in worker_node_and_gpu_ids]
         self._run_workers("update_environment_variables",
                           all_args=all_args_to_update_environment_variables)
 
         distributed_init_method = get_distributed_init_method(
             driver_ip, get_open_port())
 
-        def collect_arg_helper_func(**kwargs):
-            # avoid writing `{"name": value}` manually
-            return kwargs
-
-        init_worker_all_kwargs = []
-
         # Initialize the actual workers inside worker wrapper.
-        for rank, (node_id, _) in enumerate(worker_node_and_gpu_ids, ):
+        init_worker_all_kwargs = []
+        for rank, (node_id, _) in enumerate(worker_node_and_gpu_ids):
             local_rank = node_workers[node_id].index(rank)
-            init_worker_all_kwargs.append(
-                collect_arg_helper_func(
-                    model_config=self.model_config,
-                    parallel_config=self.parallel_config,
-                    scheduler_config=self.scheduler_config,
-                    device_config=self.device_config,
-                    cache_config=self.cache_config,
-                    load_config=self.load_config,
-                    local_rank=local_rank,
-                    rank=rank,
-                    distributed_init_method=distributed_init_method,
-                    lora_config=self.lora_config,
-                    vision_language_config=self.vision_language_config,
-                    is_driver_worker=rank == 0,
-                ))
+            init_worker_all_kwargs.append({
+                "model_config": self.model_config,
+                "parallel_config": self.parallel_config,
+                "scheduler_config": self.scheduler_config,
+                "device_config": self.device_config,
+                "cache_config": self.cache_config,
+                "load_config": self.load_config,
+                "local_rank": local_rank,
+                "rank": rank,
+                "distributed_init_method": distributed_init_method,
+                "lora_config": self.lora_config,
+                "vision_language_config": self.vision_language_config,
+                "is_driver_worker": rank == 0,
+            })
```

Comment:
I considered this, but i think keep the calling-like code is better.

Reply:
@youkaichao could you explain why? it's more code and more indirection and not any more readable...

Reply:
Perhaps subjective but imo this is actually more readable due to the spacing

Reply:
In the future I plan to use WorkerWrapper for all executors. calling-like code will be easier to modify because it is similar to existing code, but dict-like code would require many changes.

Kind of like javascript syntax suger: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer 

Maybe we should move `collect_arg_helper_func` to utils, so that in the future all executors can use this.

Reply:
OK I'll revert it. But IMHO it's not a good idea to complicate things based on ideas of future generalization, better to do that at the time. Won't it be essentially the same set of parameters used to initialize the other kinds of workers in other executors too?

Reply:
@youkaichao ok reverted